### DISCUSSION TITLE
Markup for headings and horizontal lines

### DISCUSSION THREAD

**Discussion by yann1cks** [athensresearch/athens#848] [Feature Requests]
I want to suggest to additional markup features:

1. Horizontal/separator lines: I am not using Roam but the following code [seems to be possible](https://forum.roamresearch.com/t/ability-to-do-a-horizontal-rule/1768) in Roam `(---)`
2. Support for H1, H2, H3 ...  headings. Perhaps markdown language with `# H1`, `## H2` ... could be support as well.

Both suggestions would make it much easier to style and organize knowledge within pages. But I am not sure how easy it is to implement.

**Comment by Limezy** [comment]
Hi, I made a proposal here that is kinda related
https://github.com/athensresearch/athens/discussions/902

**Comment by shanberg** [comment]
Horizontal rules are an interesting class of decorative content, unlike (afaik) every other block in Athens, which represents some thought or content, and can exist outside its context.

A more semantic way to implement HRs might be to define "breaks" in a page's content, which could then be styled with a horizontal rule, but I imagine that's more cognitive overhead than the alternative.

**Reply by jefftangx** [reply]
Breaks in markdown are deonated with triple dashes: `---`

e.g.

after break

---

before break


> can exist outside its context

Really interesting point. What if there was one universal horizontal rule. And we could see its context everywhere? It would be like if there were one universal `[[---]]` or `((---))` page or block.

I've thought about this in the context of emojis. Slack has a good emoji search. We could do something like that with emojis.

**Reply by shanberg** [reply]
I think the universal inclusion is a better case for emojis than for HRs, because HRs are meaningless in isolation, and instances are only correlated in that you're more likely to use HRs in long/organized sequences than in atomic bullet-style block sequences, whereas emojis can be used for a bunch of things, but because they _do_ carry meaning, not because they don’t.

I think HRs are pure formatting.

They could live inside normal blocks, which is fine alongside other content...but a block with only an HR is meaningless outside its context.

I think may merit a special type of block which doesn't transclude, can't become a page, etc. You'd still want to be able to drag them around though, or convert them to normal blocks if you edit the content.
